entrypoint.sh

```diff
@@ -0,0 +1,9 @@
+#!/usr/bin/env bash
+
+# Load DAGs examples (default: Yes)
+if [[ -z "$AIRFLOW__CORE__LOAD_EXAMPLES" && "${LOAD_EX:=n}" == n ]]; then
+  AIRFLOW__CORE__LOAD_EXAMPLES=False
+fi
+
+airflow scheduler &
+exec airflow webserver
```

Comment:
What is this doing and does it differ from the `CMD` bits in the Dockerfile? (bash newbie here)

Reply:
Since we need two concurrently running commands, the `&` runs the first command in the background, while the `exec` replaces the currently executing bash shell with the `airflow webserver` process. This theoretically basically replicates what 
```
CMD ["airflow", "webserver", "-D"]
CMD ["airflow", "scheduler", "-D"]
```
do, so we should only use one or the other. The advantage of using an `ENTRYPOINT` command is that we can extend it at a later point if needed.

Reply:
Great, thanks!